src/inmanta/agent/config.py

```diff
@@ -173,6 +173,31 @@
     is_lower_bounded_int(60),
 )
 
+agent_db_connection_pool_min_size: Option[int] = Option(
```

Comment:
These need a default  that is derived from the actual value of the deprecated setting, not of the default of that setting.

Reply:
Added dynamic defaults to pool size options for server and scheduler